### ISSUE TITLE
Table 'db.versions' doesn't exist: SHOW FULL FIELDS FROM `versions`

### ISSUE THREAD

**Issue by stan** [airblade/paper_trail#401]
Papertrail 3.0.3 and Rails 4.1.4

Using only custom version classes, getting `ActiveRecord::StatementInvalid (Mysql2::Error: Table 'db.versions' doesn't exist: SHOW FULL FIELDS FROM versions)` error every time a change is made to a model in development (need to restart dev env to fix), despite using the recommended initializer:

```
# config/initializers/paper_trail.rb

# the following line is required for PaperTrail >= 3.0.3 with Rails
PaperTrail::Rails::Engine.eager_load!

PaperTrail::Version.module_eval do
  self.abstract_class = true
end
```

I do not want to create a dummy versions table. Works fine in 3.0.2

**Comment by ngelx** [comment]
Hi, i'm having exactly the same issue with version 4.0.0.rc1 Rails 4.2 and ruby 2.1.2. I've set up paper_trail.rb initialiser as described on the doc, and got some custom version tables.

**Comment by stan** [comment]
Issue is back (ruby 2.1.5 / Rails 4.2.2) - fixed for me by #492
